zwave_js_server/model/controller/inclusion_and_provisioning.py

```diff
@@ -151,6 +151,7 @@ def to_dict(self) -> Dict[str, Any]:
     @classmethod
     def from_dict(cls, data: Dict[str, Any]) -> "QRProvisioningInformation":
         """Return QRProvisioningInformation from data dict."""
+        # pylint: disable=unexpected-keyword-arg
```

Comment:
Why do we need the disable?

Reply:
the new pylint version triggers an error for each of the keyword arguments. We use keyword args for readability because it's such a big list.

Reply:
I think the right long term fix is to force keyword arguments everywhere and we can remove the disable. I'll log an issue to track that

Reply:
Sounds like a pylint bug.